### PR TITLE
Update docs for MAX_FILE_SIZE_PUT_OBJECT.

### PR DESCRIPTION
**Pull request by zwj102030**
**Summary**
Update the doc for MAX_FILE_SIZE_PUT_OBJECT.

**Related PR**
https://github.com/fluent/fluent-bit/pull/5688 

Signed-off-by: Weijian Zhang <weijianz@amazon.com>

### CODE REVIEW

File: pipeline/outputs/s3.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ See [here](https://github.com/fluent/fluent-bit-docs/tree/43c4fe134611da471e706b
...
+| MAX_FILE_SIZE_PUT_OBJECT             | Max file size allowed to publish to S3 MAX: 1GB.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | 1,073,741,824 bytes                         |
```
Comment (PettitWesley) [inline pipeline/outputs/s3.md:29]: this isn't a new user config option. Its a different max size for `total_file_size` for the use_put_object case

Comment (PettitWesley) [inline pipeline/outputs/s3.md:29]: this isn't a new user config option. Its a different max size for `total_file_size` for the `use_put_object` case

**Review by PettitWesley — CHANGES_REQUESTED** [review]

File: pipeline/outputs/s3.md (hunk 1 above)
Comment (zwj102030) [reply pipeline/outputs/s3.md:29]: so we may dont need the `MAX_FILE_SIZE_PUT_OBJECT` to be list in the documentation .

Comment (PettitWesley) [reply pipeline/outputs/s3.md:29]: yea just note the max allowed value in the `total_file_size` row a little farther up, that's what I mean

**Comment by PettitWesley** [comment]
@zwj102030 please rebase and update

**Comment by PettitWesley** [comment]
Closing in favor of: https://github.com/PettitWesley/fluent-bit-docs/pull/new/s3-putobject-maxsize

**Pull request closed without merging**